csrc/adam/cpu_adam.cpp

```diff
@@ -672,11 +672,21 @@ int ds_adam_step_plus_copy(int optimizer_id,
     return 0;
 }
 
+int destroy_adam_optimizer(int optimizer_id)
+{
+    // std::cout << "Adam Optimizer #" << optimizer_id
+    //          << " is destroyed." << std::endl;

```

Comment:
Do we want to give this warning? or just simply remove it

Reply:
We don't really need it, other than for debug, I thought to leave it while we are sorting out if I proposed the right approach and will delete it just before this is approved and then clean it up.

Reply:
I removed it.

Reply:
Thanks